### DISCUSSION TITLE
Interrupted transitions

### DISCUSSION THREAD

**Discussion by TreetopFlyer** [alpinejs/alpine#641] [4. General]
Sorry if this has been asked or if I misread the docs.
In the following example

    <div x-data="{open:false}">
     <button x-on:click="open = !open">Open</button>
     <p>Immediate: <span x-text="open"></span></p>
     <p>Transition: <span x-text="open" x-show.transition.duration.500ms="open"></span></p>
    </div>

If I click three times very fast (before the transition finishes) on the button, the "immediate" span will always show the correct state, but the "transition" span will not be able to correctly keep up with the changing value and actually ends incorrectly. Is there a way to resolve this without getting rid of potentially long transitions?

**Comment by SimoTod** [comment]
There's a pending PR which should address this issue and should be merged in the next release.

**Reply by SimoTod** [reply]
Can you try https://cdn.jsdelivr.net/gh/SimoTod/alpine@ca821803fac18fe73794fdd93fd76ce6858911e8/dist/alpine.min.js and let me know if it fixes the problem?

**Reply by TreetopFlyer** [reply]
@SimoTod ty for quick reply, amazing works like a charm!

**Reply by SimoTod** [reply]
Cool. It should be in the next release. 🤞 Watch this space
